[2025-05-26 23:26:12 +0200] [219902] [INFO] Starting gunicorn 23.0.0
[2025-05-26 23:26:12 +0200] [219902] [INFO] Listening at: http://0.0.0.0:8000 (219902)
[2025-05-26 23:26:12 +0200] [219902] [INFO] Using worker: sync
[2025-05-26 23:26:12 +0200] [219916] [INFO] Booting worker with pid: 219916
[2025-05-27 00:31:00 +0200] [219902] [CRITICAL] WORKER TIMEOUT (pid:219916)
[2025-05-27 00:31:00 +0200] [219916] [ERROR] Error handling request (no URI read)
Traceback (most recent call last):
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/workers/sync.py", line 133, in handle
    req = next(parser)
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/http/parser.py", line 41, in __next__
    self.mesg = self.mesg_class(self.cfg, self.unreader, self.source_addr, self.req_count)
                ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/http/message.py", line 259, in __init__
    super().__init__(cfg, unreader, peer_addr)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/http/message.py", line 60, in __init__
    unused = self.parse(self.unreader)
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/http/message.py", line 274, in parse
    line, rbuf = self.read_line(unreader, buf, self.limit_request_line)
                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/http/message.py", line 326, in read_line
    self.get_data(unreader, buf)
    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/http/message.py", line 262, in get_data
    data = unreader.read()
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/http/unreader.py", line 36, in read
    d = self.chunk()
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/http/unreader.py", line 63, in chunk
    return self.sock.recv(self.mxchunk)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "/home/markmur88/Documentos/Entorno/envAPP/lib/python3.13/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
    sys.exit(1)
    ~~~~~~~~^^^
SystemExit: 1
[2025-05-27 00:31:00 +0200] [219916] [INFO] Worker exiting (pid: 219916)
[2025-05-27 00:31:01 +0200] [219902] [ERROR] Worker (pid:219916) was sent SIGKILL! Perhaps out of memory?
[2025-05-27 00:31:01 +0200] [377313] [INFO] Booting worker with pid: 377313
